.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
```

Comment:
let's not do a fix on save. in my experience, it gets confusing.

Reply:
I will switch it to a runAll or something similar